client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -105,6 +126,28 @@ export function SitesDashboard( { queryParams: { search, status = 'all' } }: Sit
 							{ displayMode === 'tile' && (
 								<SitesGrid isLoading={ isLoading } sites={ filteredSites } />
 							) }
+							{ selectedStatus.hiddenCount && (
```

Comment:
> Here we need to check if the `hiddenCount` is greater than zero. We need to do that because despite `0` being a falsy value, React interprets it as literal `0` and displays it:

Learned something new! And obvious in hindsight 😄 

Fixed with 4dbb235570500a3d831ca587672350fcf33a4d4e